R/later_recurring.R

```diff
@@ -0,0 +1,24 @@
+#' @describeIn later Schedules a recurring task
+#' @param limit Number of times to repeat the function. If `NA` (the default)
+#'   then no limit.
+#' @examples
+#' later_recurring(~cat("Hello from the past\n"), 3, limit = 2)
```

Comment:
Add (untested) example of preempt functionality. Also reduce wait time for existing example

```suggestion
#' # Limit number of executions to 3
#' later_recurring(~cat("Hello from the past\n"), 1, limit = 3)
#' 
#' # Stop recurring when the return value is `FALSE`
#' later_recurring(function() {
#'   message("Flipping a coin to see if we run again...")
#'   sample(c(T, F), 1)
#' }, 0.25, limit = NA)
```